persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/ClinicalDataMapper.xml

```diff
@@ -214,4 +221,37 @@
         GROUP BY clinical_patient.ATTR_ID, clinical_patient.ATTR_VALUE
     </select>
 
+    <select id="getSampleClinicalDataClinicalTable" resultType="org.cbioportal.model.ClinicalData">
+        SELECT
+        <include refid="selectSample">
+            <property name="prefix" value=""/>
+        </include>
+        <include refid="fromSample"/>
+        <include refid="whereSampleSearchAndPagination"/>
```

Comment:
Huh - I wouldn't have thought to do it this way. I like how this allows you to reuse the select / from logic.